app/AppDelegate.m

```diff
@@ -5,6 +5,8 @@
 //  Created by Theodore Dubois on 10/17/17.
 //
 
+#import <Foundation/Foundation.h>
```

Comment:
Since this file imports AppDelegate.h, and by extension UIKit/UIKit.h, I don't think this is necessary?

Reply:
true.. I remove it